lab-michael/model/recipe.js

```diff
@@ -2,12 +2,13 @@
 
 const uuid = require('node-uuid');
 
-module.exports = function(name, content, mealType) {
+const Recipe = function(name, content, mealType) {
```

Comment:
Why change this? It's still the only function in this module, so no need to provide it a name and then export the constructor.